de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/ArchiveIncomingReferencePointNegotiation.java

```diff
@@ -64,30 +65,32 @@ public ArchiveIncomingProjectNegotiation(
 
     @Override
     protected void transfer(IProgressMonitor monitor,
-        Map<String, IProject> projectMapping, List<FileList> missingFiles)
-        throws IOException, SarosCancellationException {
+        Map<String, IReferencePoint> referencePointMapping,
+        List<FileList> missingFiles) throws IOException,
+        SarosCancellationException {
 
         awaitActivityQueueingActivation(monitor);
         monitor.subTask("");
 
         /*
-         * the user who sends this ProjectNegotiation is now responsible for the
-         * resources of the contained projects
+         * the user who sends this ReferencePointNegotiation is now responsible
+         * for the resources of the contained referencePoint
          */
-        for (Entry<String, IProject> entry : projectMapping.entrySet()) {
+        for (Entry<String, IReferencePoint> entry : referencePointMapping
+            .entrySet()) {
 
-            final String projectID = entry.getKey();
-            final IProject project = entry.getValue();
+            final String referencePointID = entry.getKey();
+            final IReferencePoint referencePoint = entry.getValue();
             /*
-             * TODO Queuing responsibility should be moved to Project
+             * TODO Queuing responsibility should be moved to ReferencePoint
              * Negotiation, since its the only consumer of queuing
              * functionality. This will enable a specific Queuing mechanism per
              * TransferType (see github issue #137).
              */
-            referencePointManager.put(project.getReferencePoint(), project);
-            session.addReferencePointMapping(projectID,
-                project.getReferencePoint());
-            session.enableQueuing(project.getReferencePoint());
+            // referencePointManager.put(referencePoint.getReferencePoint(),
+            // referencePoint);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMeiDZywvPe4LUfQ&open=AWZfKMeiDZywvPe4LUfQ)

[ISSUE_KEY]: AWZfKMeiDZywvPe4LUfQ